components/purchase_modal/purchase_modal.tsx

```diff
@@ -207,13 +217,48 @@ export default class PurchaseModal extends React.PureComponent<Props, State> {
                     id='list-plans-radio-buttons'
                     values={options!}
                     value={this.state.selectedProduct?.id as string}
-                    badge={{matchVal: this.state.currentProduct?.id as string, text: badgeTitle}}
+                    sideLegend={{matchVal: this.state.currentProduct?.id as string, text: sideLegendTitle}}
                     onChange={(e: any) => this.onPlanSelected(e)}
                 />
             </div>
         );
     }
 
+    displayDecimals = () => {
+        const price = this.state.selectedProduct?.price_per_seat.toFixed(2);
+        let decimals = null;
+        if (this.state.selectedProduct?.billing_scheme === 'flat_fee') {
+            decimals = (price || 1 % 1).toString().substring(2);
+        }
```

Comment:
Same comments as Nick mentioned in: https://github.com/mattermost/customer-web-server/pull/511/files#r664611042 here and other occurences of `flat_fee` string below